mock/server/server.go

```diff
@@ -128,12 +129,14 @@ func FakeHandler() http.Handler {
 	e.GET("/api/v1/workers/:worker", getWorker)
 	e.POST("/api/v1/workers", addWorker)
 	e.PUT("/api/v1/workers/:worker", updateWorker)
+	e.POST("/api/v1/workers/:worker/refresh", refreshWorkerAuth)
 	e.DELETE("/api/v1/workers/:worker", removeWorker)
 
 	// mock endpoints for authentication calls
 	e.GET("/token-refresh", getTokenRefresh)
 	e.GET("/authenticate", getAuthenticate)
 	e.POST("/authenticate/token", getAuthenticateFromToken)
+	e.POST("/validate-token", validateToken)
```

Comment:
should match the handler
```suggestion
	e.GET("/validate-token", validateToken)
```